app/presenters/prior_authority/check_answers/report.rb

```diff
@@ -57,11 +57,16 @@ def case_and_hearing_detail_cards
       end
 
       def about_request_section
-        [
+        base = [
           PrimaryQuoteCard.new(application, verbose: @verbose),
           AlternativeQuotesCard.new(application, verbose: @verbose),
           ReasonWhyCard.new(application),
         ]
+        if @verbose
+          base + application.further_informations.map { CompactFurtherInformationCard.new(_1) }
+        else
+          base # TODO: CRM457-1226: Add card for most recent further information
+        end
```

Comment:
I am using a method to extract logic

```
      def about_request_section
        [
          PrimaryQuoteCard.new(application, verbose: @verbose),
          AlternativeQuotesCard.new(application, verbose: @verbose),
          ReasonWhyCard.new(application),
          further_information_card,
        ].compact
      end

      private

      def further_information_card
        return unless application.further_informations.any?

        FurtherInformationCard.new(application)
      end
```


this could be extended with your logic after the fact fairly easily i imagine. I am just working on tests atm.

```
      def further_information_card
        return unless application.further_informations.any?
        
        if @verbose
           application.further_informations.map { CompactFurtherInformationCard.new(_1) }
        else
           FurtherInformationCard.new(application)
         end
      end
```

Reply:
Cool. I might need to wait until  #666 is merged and then rebase off that, otherwise my PR will just get in the way.